docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -346,10 +363,10 @@ class ComponentExample extends React.PureComponent<IComponentExampleProps, IComp
   private getDisplayName = () => this.props.examplePath.split('/')[1]
 
   private renderWithProvider(ExampleComponent) {
-    const { showRtl, theme } = this.state
+    const { showRtl, componentVariables } = this.state
 
     const newTheme: IThemeInput = {
-      componentVariables: theme.componentVariables,
+      componentVariables,
```

Comment:
this means that `theme` is only about styles that are provided by CSS-like objects. At the same time, it is possible for theme to provide static CSS styles, and those won't be applied with current implementation. Could you, please, suggest what is our vision for this question?